### PR TITLE
feat: add Divan's Powder Coating and Polarvoid

### PR DESCRIPTION
**Pull request by DuckySoLucky**
## Description

Adds https://wiki.hypixel.net/Polarvoid_Book and https://wiki.hypixel.net/Divan%27s_Powder_Coating
> Not sure about the applicationWorth values

## Preview

image

### CODE REVIEW

File: calculators/itemCalculator.js [hunk 1]
```diff
@@ -503,6 +503,30 @@ const calculateItem = (item, prices, returnItemData) => {
...
+        count: ExtraAttributes.polarvoid,
+      };
+      price += calculationData.price;
+      calculation.push(calculationData);
+    }
+
+    // DIVAN'S POWDER COATING
+    if (ExtraAttributes.divan_powder_coating) {
+      const calculationData = {
+        id: 'divan_powder_coating',
```
Comment (Altpapier) [inline calculators/itemCalculator.js]: uppercase

**Review by Altpapier — CHANGES_REQUESTED** [review]
Will also need to ask matt what he thinks are good application values (@MattTheCuber)

**Comment by MattTheCuber** [comment]
I think `1` is good for Polarvoid Book as it is similar in usefulness and price to Hot Potato Books (which is also at 1).

Divan's Powder Coating is a tricky one, I think to start maybe `0.8`? Just a vibe guess because its pricey.

File: constants/applicationWorth.js [hunk 2]
```diff
@@ -31,6 +31,8 @@ const applicationWorth = {
   prestigeItem: 1,
   reforge: 1,
   winningBid: 1,
+  polarvoid: 1,
+  divanPowderCoating: 1,
```
Comment (MattTheCuber) [inline constants/applicationWorth.js]: I try to keep these grouped by value and then sorted alphabetical within each group.

**Review by MattTheCuber — CHANGES_REQUESTED** [review]
Also need to update README I believe (for application worths)

File: constants/applicationWorth.js (hunk 2 above)
Comment (DuckySoLucky) [reply constants/applicationWorth.js]: Not entirely sure how you group it, I tried to guess it. If it's not correct just create a commit suggestion

**Review by MattTheCuber — APPROVED** [review]

**Review by Altpapier — APPROVED** [review]

**Pull request merged**